change/base.js

```diff
@@ -2,6 +2,20 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-  $("#time").text( Date.now() );
+  $("#time").text( Date.now() + " yr mom" );
 
-})
+  // sets all values to 0
+  $("input").val(0);
+  
+  $("input").on("change", function doMath(l, r){
```

Comment:
What is passed to the callback is predetermined by the function that calls it (in this case 'on').  doMath(event) or doMath(e) would make more sense